esx_service/utils/auth_api.py

```diff
@@ -432,13 +454,24 @@ def _tenant_vm_add(name, vm_list):
         error_info = error_code.generate_error_info(ErrorCode.VM_ALREADY_IN_TENANT, vm_list, name)
         return error_info
 
+    error_info, tenant_list = get_tenant_list_from_db()
```

Comment:
+1 on this. Function "any_vm_part_of_another_tenant" does not need to pass in param @tenant_list.